config.yaml

```diff
@@ -29,7 +29,8 @@ developer.mode.enabled: false
 # Authentication configuration
 # -----------------------------
 
-token.privatekey : `-----BEGIN RSA PRIVATE KEY-----
+token.privatekey : > 
```

Comment:
I had missed out the on the indentation. Starting to miss Python ;)
Fixed here:
https://github.com/sbose78/almighty-core/blob/5c5bab971cf052e2464b1f0827b19a747db9797d/config.yaml#L32